clouddriver-web/src/main/groovy/com/netflix/spinnaker/clouddriver/controllers/RawResourceController.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2020 Coveo, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.clouddriver.controllers;
+
+import com.netflix.spinnaker.clouddriver.model.RawResource;
+import com.netflix.spinnaker.clouddriver.model.RawResourceProvider;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+import java.util.stream.Collectors;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.MediaType;
+import org.springframework.security.access.prepost.PostAuthorize;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping(produces = MediaType.APPLICATION_JSON_VALUE)
+class RawResourceController {
+
+  @Autowired(required = false)
+  List<RawResourceProvider> rawResourceProviders = new ArrayList<>();
```

Comment:
Use constructor injection.

Reply:
Also, as this is a `List` type, `required = false` is unnecessary, as Spring will auto-populate an empty list if no beans are found implementing this type.